generate/seccomp/syscall_compare.go

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
```

Comment:
This comment doesn't fit recommended style. See https://golang.org/doc/effective_go.html#commentary.